src/test/java/com/znsio/teswiz/screen/android/vodqa/VodqaScreenAndroid.java

```diff
@@ -129,4 +130,18 @@ public VodqaScreen swipeByPassingPercentageAttributes(int atPercentScreenHeight,
         driver.swipeByPassingPercentageAttributes(atPercentScreenHeight, fromPercentageWidth, toPercentScreenWidth);
         return this;
     }
+
+
+    @Override
+    public VodqaScreen putAppInTheBackground(int time) {
+        driver.putAppInBackground(time);
+        visually.checkWindow(SCREEN_NAME, "Home screen after putting app in background");
+        return this;
+    }
+
+    public boolean validateAppWorkInBackground() {
+        LOGGER.info("Validating current app package to know app work in background");
+        String getPackageDetails = ((AndroidDriver) driver.getInnerDriver()).getCapabilities().getCapability("appPackage").toString();
```

Comment:
how does this comparison ensure the app is in the background?

Reply:
done